src/PIL/PcfFontFile.py

```diff
@@ -206,9 +219,11 @@ def _load_bitmaps(self, metrics):
             mode = "1"
 
         for i in range(nbitmaps):
-            x, y, l, r, w, a, d, f = metrics[i]
-            b, e = offsets[i], offsets[i + 1]
-            bitmaps.append(Image.frombytes("1", (x, y), data[b:e], "raw", mode, pad(x)))
+            left, right = metrics[i][:2]
+            b, e = offsets[i : i + 2]
+            bitmaps.append(
+                Image.frombytes("1", (left, right), data[b:e], "raw", mode, pad(left))
```

Comment:
```suggestion
            xsize, ysize = metrics[i][:2]
            b, e = offsets[i : i + 2]
            bitmaps.append(
                Image.frombytes("1", (xsize, ysize), data[b:e], "raw", mode, pad(xsize))
```
Wait, sorry, I think I got confused in my PR. This corrects it.